workspaces/diff-engine/src/lib.rs

```diff
@@ -1,7 +1,7 @@
 #![allow(dead_code, unused_imports, unused_variables)]
 
 mod commands;
-mod events;
+pub mod events;
 mod interactions;
```

Comment:
I wondered if we could be more surgical than exposing every single event struct, bu thinking about it I think you're right about just doing the whole lot. Commands and Events are the I/O types to getting anything accomplished, so we might as well accept that and move on.

Reply:
@JaapRood so Rust question: I had to make this module public for _anything_ inside it to be accessible. My impression is that you still have to mark structs/traits as `pub` for them to be public.

Is there a way to surgically do that without making it public?

Reply:
There is! What I've seen in other Rust codebases and adopted for `diff-engine` is the approach of declaring inline modules and re-exporting: https://github.com/opticdev/optic/blob/develop/workspaces/diff-engine/src/lib.rs#L29-L34.